### ISSUE TITLE
An action could not be found at the URI

### ISSUE THREAD

**Issue by moltar** [miyajan/npm-audit-pr#3]
Any idea what could be causing this?

```
Current runner version: '2.263.0'
Operating System
Virtual Environment
Prepare workflow directory
Prepare all required actions
Download action repository 'actions/checkout@v2.0.0'
Download action repository 'miyajan/npm-audit-fix@v1.0.0'
##[error]An action could not be found at the URI 'https://api.github.com/repos/miyajan/npm-audit-fix/tarball/v1.0.0'
```

Workflow config:

```yml
jobs:
  npm-audit-pr:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2.0.0
      - uses: miyajan/npm-audit-fix@v1.0.0
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by miyajan** [comment]
@moltar 
Sorry, the name of action was wrong in the README.md. I have fixed it.

And I have released v1.0.1 to fix some bugs. Please specify `miyajan/npm-audit-pr@v1.0.1`.

**Comment by moltar** [comment]
thanks! :)
